params/config.go

```diff
@@ -187,7 +191,10 @@ func (c *XDPoSConfig) String() string {
 ConsensusVersion will return the consensus version to use for the provided block number. The returned int represent its version
 TODO: It's a dummy value for now until the 2.0 consensus engine is fully implemented.
 */
-func BlockConsensusVersion(num *big.Int) int {
+func (c *XDPoSConfig) BlockConsensusVersion(num *big.Int) int {
+	if c.V2ConsensusBlockNumber != nil && num.Cmp(c.V2ConsensusBlockNumber) > 0 {
+		return 2
+	}
 	return 1
```

Comment:
I think we can make these `1` and `2`, and const variable. 
Also, in the XDPoS.go, we can reassign variable. Then we don't need to always do package.variable
```

const (
	V1 = params.V1
)

```